src/components/button-group/ButtonGroup.scss

```diff
@@ -3,19 +3,19 @@
 /**************************************
  * Button Group
  **************************************/
-.btn-group {
+.bdl-ButtonGroup {
     position: relative;
 
     .toggle-overlay {
         display: inline;
 
-        > .btn {
+        > .bdl-Button {
             margin-left: 0;
             padding-right: 11px;
             padding-left: 11px;
             border-radius: 0 4px 4px 0;
 
-            &.btn-primary {
+            &.bdl-PrimaryButton {
```

Comment:
It looks like PrimaryButton decorates the `Button` component classes, which indicates that it should be a modifier class, such as `bdl-Button--primary`.

Reply:
+1 to that - they are standalone components at the React level but I think we should consider  changing it to `<Button primary />` or something like that

Reply:
Yup, I agree that we should eventually merge `Button` and `PrimaryButton`. For now, I'll just change the class name to `.bdl-Button--primary`, since we'll likely be using that class even after the components get merged.